x-pack/plugins/snapshot_restore/public/application/sections/home/restore_list/restore_list.tsx

```diff
@@ -63,12 +64,18 @@ export const RestoreList: React.FunctionComponent = () => {
   } = useLoadRestores(currentInterval);
 
   const { uiMetricService, history } = useServices();
+  const { core } = useAppContext();
 
   // Track component loaded
   useEffect(() => {
     uiMetricService.trackUiMetric(UIM_RESTORE_LIST_LOAD);
   }, [uiMetricService]);
 
+  useExecutionContext(core.executionContext, {
+    type: 'application',
+    page: 'snapshotRestoreRestores',
```

Comment:
this one reads a bit funny, what do you think about `snapshotRestoreList` or `snapshotRestoreRestoreList` 🤔

Reply:
What do you think about `snapshotRestoreRestoreTab`? I think I originally had "list", but changed it since it actually covers any requests within the tab (not just the list view).

Reply:
Yeah I think `snapshotRestoreRestoreTab` sounds a lot better!